### PR TITLE
ROX-15851: Clarify onscreen hint for runtime policies

### PR DESCRIPTION
**Pull request by vjwilson**
## Description

Low-hanging fruit to help with this concern:
> Clearly indicate when combinations of policy criteria don't make sense or a policy is invalid because of the criteria selection and why?

## Checklist
- [ ] Investigated and inspected CI test results

## Testing Performed

Multiple stakeholders (UI, UX, docs, API) to proofread
Screen Shot 2023-03-10 at 11 39 14 AM

### CODE REVIEW

**Review by clickboo — APPROVED** [review]
Lgtm, but I'd like for @kcarmichael08 to approve the wording.

File: ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx [hunk 1]
```diff
@@ -142,20 +142,21 @@ function PolicyBehaviorForm() {
...
-                        <div>
+                        <p>
                             Build-time policies apply to image fields such as CVEs and Dockerfile
                             instructions.
-                        </div>
-                        <div>
+                        </p>
+                        <p>
                             Deploy-time policies can include all build-time policy criteria but they
                             can also include data form your cluster configurations, such as running
```
Comment (kcarmichael08) [inline ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx:151]: form = from?

File: ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx [hunk 2]
```diff
@@ -142,20 +142,21 @@ function PolicyBehaviorForm() {
...
-                        </div>
-                        <div>
+                        </p>
+                        <p>
                             Runtime policies can include all build-time and deploy-time policy
-                            criteria but they can also include data about process executions during
-                            runtime.
-                        </div>
+                            criteria but they <strong>must</strong> include one policy criterion
+                            from either process, network flow, audit log events, or Kubernetes
```
Comment (kcarmichael08) [inline ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx:157]: either is used when you have two choices so would remove that word

**Comment by kcarmichael08** [comment]
Sorry, couldn't figure out how to suggest inline, but had 2 comments. Otherwise looks great!

**Pull request merged**